WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -7,9 +7,8 @@ import class Yosemite.Store
 
 protocol PointOfSaleItemsControllerProtocol {
     var itemsViewStatePublisher: any Publisher<ItemsViewState, Never> { get }
-    func loadInitialItems(base: ItemListBaseItem) async
+    func reloadItems(base: ItemListBaseItem) async
```

Comment:
```suggestion
    func loadFirstPageOfItems(base: ItemListBaseItem) async
```
Or just `loadItems`? Not a big deal, but it might feel more intuitive at the call site to use a "load" method in the refresh function, than a "reload" method in the initial load function.

Reply:
My first change was actually naming it `loadItems`, then renamed it to `reloadItems` just because the "load" doesn't seem to indicate that it resets any previous states. But I think if the protocol just has `load*` + `loadNext*` it's pretty obvious one is to load the first set and the other loads the next. Now that you also suggest `loadItems` let's name it this way. Updated in https://github.com/woocommerce/woocommerce-ios/pull/14852/commits/8e3fa666a743e7ae0076720d609b9fe170159d3d.